services/styles.js

```diff
@@ -23,6 +23,7 @@ var Styles = {};
  * @param {string} [config.ownerId]
  * @param {boolean} [config.metadata] - If true, `mapbox:` specific metadata will be preserved
  * @param {boolean} [config.draft=false] - If `true` will retrieve the draft style, otherwise will retrieve the published style.
+ * @param {boolean} [config.fresh=false] - If `true` will bypass the style cache, with a lower rate limit than cached requests.
```

Comment:
I think it would be really helpful to future users to include some version of @samanpwbb's disclaimer in the JSDoc. What do you think about something like this? 


```suggestion
 * @param {boolean} [config.fresh=false] - If `true`, will bypass the cached version of the style. Fresh style requests have a lower rate limit than cached requests and may have a higher latency. `fresh=true` should never be used in production or high concurrency environments.
```

Reply:
Just on this wording I think just saying not for use in high concurrency environments is enough, we could drop the "in production" wording. One of the common use cases of this SDK is to mutate styles within your own Mapbox account in which case you often need to get the current resource. eg. imagine if Mapbox Studio didn't use fresh=true, so for some production applications fresh=true is absolutely the right thing to do.